regtest.sh

```diff
@@ -171,92 +169,73 @@ _run-one-instance() {
   local instance_dir=$case_dir/$test_instance
   mkdir --verbose -p $instance_dir
 
-  # NOTE: This is a nested case statement on the same variable ($impl).  If
-  # it's fast_counts, we just run gen_counts.R.  If it's anything else, then we
-  # generate raw reports, run them through the appropriate client, and then sum
-  # the bits.
+  banner "Generating reports (gen_reports.R)"
+
+  # the TRUE_VALUES_PATH environment variable can be used to avoid
+  # generating new values every time.  NOTE: You are responsible for making
+  # sure the params match!
+
+  local true_values=${TRUE_VALUES_PATH:-}
+  if test -z "$true_values"; then
+    true_values=$instance_dir/case_true_values.csv
+    tests/gen_true_values.R $distr $num_unique_values $num_clients \
+                            $values_per_client $num_cohorts \
+                            $true_values
+  else
+    # TEMP hack: Make it visible to plot.
+    # TODO: Fix compare_dist.R
+    ln -s -f --verbose \
+      $PWD/$true_values \
+      $instance_dir/case_true_values.csv
+  fi
 
   case $impl in
-    fast_counts)
```

Comment:
I think you deleted fast_counts, which is fine with me, but mention it in the pull request description